internal/testhelper/testhelper.go

```diff
@@ -168,11 +167,10 @@ func SetupTestGRPCServer(tb testing.TB, logger golog.Logger) (*grpc.Server, int)
 	listener, err := net.Listen("tcp", ":0")
 	test.That(tb, err, test.ShouldBeNil)
 	grpcServer := grpc.NewServer()
-	goutils.PanicCapturingGo(func() {
```

Comment:
we made this change when moving this to a separate package because there is a lint rule that prevents public functions from not handling return values (i.e., the error return value of `grpcServer.Serve`)